Statsman/Core/TimeSplitting/PeriodStatisticsService.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Statsman.Core.TimeSplitting.Models;
+using Statsman.Core.TimeSplitting.Services;
+using Statsman.Models;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Watchman.Common.Models;
+using Watchman.Cqrs;
+using Watchman.DomainModel.Messages;
+using Watchman.DomainModel.Messages.Queries;
+using Watchman.Integrations.Quickchart;
+
+namespace Statsman.Core.TimeSplitting
+{
+    public partial class PeriodStatisticsService
+    {
+        private readonly IQueryBus queryBus;
+        private readonly ChartsService _chartsService;
+        private readonly StatisticsGroupingService _statisticsGroupingService;
+
+        public PeriodStatisticsService(IQueryBus queryBus, ChartsService chartsService, StatisticsGroupingService statisticsGroupingService)
+        {
+            this.queryBus = queryBus;
+            this._chartsService = chartsService;
+            this._statisticsGroupingService = statisticsGroupingService;
+        }
+
+        public Task<(Stream Chart, ResultMessage Message)> PerMinute(StatisticsRequest request)
+        {
+            return this.GetResult(request, DetailedPeriod.Minute, DateTime.UtcNow);
```

Comment:
tak się zastanawiam czy kiedyś tutaj DateTime.UtcNow nie będzie problemem
bo w przyszłości każdy serwer ma mieć możliwość ustawienia swojej strefy czasowej
ale jeśli zachowa się w całych statystykach tą samą konwencję to chyba nie powinno być problemu
no tylko trzeba uważać jak się wyświetla, to żeby w czasie lokalnym a nie UTC, ale to chyba gdzieś widziałem że miałeś dobrze zrobione

Reply:
no tylko obiekt DateTime trzyma w sobie informacje o strefie czasowej
obiekt w bazie danych to samo

Reply:
teoretycznie można zrobić tak żeby było w lokalnej strefie a nie z przesunięciem